app/controllers/notification.js

```diff
@@ -0,0 +1,38 @@
+const HANDLER = require('../utils/response-helper')
+const HttpStatus = require('http-status-codes')
+const Notifications = require('../models/Notifications')
+const helper = require('../utils/paginate')
+const User = require('../models/User')
+
+module.exports = {
+  // GET ALL THE NOTIFICATIONS FOR ALL
+  getOrgNotifications: async (req, res, next) => {
+    try {
+      const notifications = await Notifications.find({}, {}, helper.paginate(req))
+        .lean()
+        .sort({ createdAt: -1 })
+        .exec()
+      return res.status(HttpStatus.OK).json({ notifications })
+    } catch (error) {
+      HANDLER.handleError(res, error)
+    }
+  },
+  // GET LOGGED IN USER NOTIFICATIONS
+  getUserNotification: async (req, res, next) => {
+    const userId = req.user._id
+    try {
+      const user = await User.findById(userId)
+      if (!user) {
+        return res.status(HttpStatus.BAD_REQUEST).json({ msg: 'No such user exists!' })
+      }
+      // get all notifications of existing user
+      const notifications = user.notifications
+      if (notifications.length === 0) {
+        return res.status(HttpStatus.OK).json({ msg: 'No new notifications!' })
+      }
+      return res.status(HttpStatus.OK).json({ notifications })
+    } catch (error) {
+      HANDLER.handleError(res, error)
+    }
```

Comment:
if you want to block/mute notifications for certain people how will you handle it?

Reply:
As Donut is a platform for the whole organization, so I think that would be controlled by admin. So if admin disables the notification then no one will receive the notification.
I am not sure if we should do that for a particular user basis.

cc @devesh-verma

Reply:
Just add a filter list. for exceptions